README.md

```diff
@@ -102,3 +102,26 @@ Local JMX is currently enabled on the project. In order to utilize it please fol
 6. Find the health endpoint and click on it to view its attributes and operations. You can use the attributes and operations to check the health of the login-gateway and perform management tasks.
 You can now use JConsole to monitor and manage your local application by accessing the available endpoints.
 Remote JMX is also an option and may be enabled with some config changes in the application.properties file.
+
+## Info endpoint
+Springboot Actuator is enabled for this project. This enables an Info endpoint that can be populated with various information that may be
+useful to the troubleshooting and usage of the application. The endpoint can be accessed via the following url:  `http://localhost:port/actuator/info`.
+The information types available are as follows:
+
+| ID     | Usage                                                        |
+|--------|--------------------------------------------------------------|
+ | build  | Exposes build information                                    |
+| env    | Exposes any property marked under "info" in the config       |
+ | git    | Exposes git related information from the git.properties file |
+| java   | Exposes Java runtime information                             |
+
+In the given example config ("example.application.yaml"), the env and git properties are enabled. 
+An example git.properties file has also been included ("example.git.properties"), simply rename it to "git.properties" in order to make use of it.
```

Comment:
This works. But how do you envision the process actually expected to work - where is the `git.properties` file supposed to come from? Should it be included internally - when the project is ran from source or externally when released? or both?

Reply:
Added tooling for user to generate git.properties file. Maven and Gradle have plugins that handle this automatically but couldn't find a solution for SBT, so just made one myself.